src/cascadia/TerminalControl/TermControl.h

```diff
@@ -26,6 +26,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         TermControl(IControlSettings settings, TerminalConnection::ITerminalConnection connection);
 
         winrt::fire_and_forget UpdateSettings();
+        winrt::fire_and_forget UpdateAppearance(IControlAppearance newAppearance);

```

Comment:
```suggestion
        winrt::fire_and_forget UpdateAppearance(const IControlAppearance& newAppearance);
```

Reply:
Adding the `&` causes a read access violation (on `add_ref` when `_UpdateAppearanceFromUIThread` is called), is it because this is a `fire_and_forget`?

Reply:
OH NO YOU'RE RIGHT KEEP IT LIKE THIS

Reply:
(just to clarify, we still want the `const` right? just not the `&`)

Reply:
(yes))